### PR TITLE
PR openapi-to-postman as package

### PR DESCRIPTION
**Pull request by thim81**
Implemented openapi-to-postman as a Service instead of using execShellCommand

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -187,21 +186,38 @@ require('dotenv').config()
...
+    testsuiteFile: testSuiteConfigFile,
+    testFlag: tmpCollectionFile
   }
+  const collectionGenerated = await oaToPostman.convert(openApiSpec, oaToPostmanConfig)
+    .catch(function(err) {
+      console.log('error: ', err)
+      throw new Error(`Collection generation failed.`)
+    })
+
+  // const collectionGenerated = await execShellCommand(
```
Comment (nicklloyd) [inline src/index.ts:206]: remove

File: src/index.ts [hunk 2]
```diff
@@ -187,21 +186,38 @@ require('dotenv').config()
...
+  // if (!collectionGenerated) {
+  //   throw new Error(`Collection generation failed.`)
+  // }
 
   // --- Portman - load generated Postman collection
   let collectionJson = {}
   try {
-    collectionJson = require(`${tmpCollectionFile}`) as CollectionDefinition
+    collectionJson = collectionGenerated as CollectionDefinition
+    // collectionJson = require(`${tmpCollectionFile}`) as CollectionDefinition
```
Comment (nicklloyd) [inline src/index.ts:220]: remove

File: src/application/OpenApiToPostmanService.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      // apply options from config file if present
+      if (options.configFile) {
+        let configFile = path.resolve(options.configFile)
+        // console.log('Options Config file: ', configFile)
+        let configFileOptions = JSON.parse(fs.readFileSync(configFile, 'utf8'))
+        options = Object.assign({}, options, configFileOptions)
+      }
+
+      // override options provided via cli
+      // if (definedOptions && !_.isEmpty(definedOptions)) {
```
Comment (nicklloyd) [inline src/application/OpenApiToPostmanService.ts:28]: remove if not needed

**Review by nicklloyd — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (thim81) [reply src/index.ts]: Done

File: src/application/OpenApiToPostmanService.ts (hunk 3 above)
Comment (thim81) [reply src/application/OpenApiToPostmanService.ts]: Done

File: src/index.ts (hunk 2 above)
Comment (thim81) [reply src/index.ts]: Done

**Pull request merged**